app/assets/stylesheets/app/public_dashboard.scss

```diff
@@ -0,0 +1,54 @@
+$pd-card-width: 40%;
+$pd-card-margin: $pd-card-width / 8;
+
+.pd-card {
+  @extend .card;
+  display: flex;
+  flex-direction: column;
+  margin: $pd-card-margin 0 0 $pd-card-margin;
```

Comment:
Shorthands of length `4` are not allowed. Value was `$pd-card-margin 0 0 $pd-card-margin`